files/src/utils/test-helpers/test-helper.ts

```diff
@@ -0,0 +1,7 @@
+import { setApp, start } from '@glimmer/test-helpers';
+import App from '../../main';
+
+QUnit.config.autostart = false;
```

Comment:
Making fake modules is a bit harder for us here, due to the fact that we are embracing "standard JavaScript" things and all modules need to be resolvable in node_modules (or we get annoying typescript and rollup errors...